demo/runners/agent_container.py

```diff
@@ -96,8 +96,23 @@ def connection_ready(self):
         return self._connection_ready.done() and self._connection_ready.result()
 
     async def handle_oob_invitation(self, message):
+        print("handle_oob_invitation()")
         pass
 
+    async def handle_connection_reuse(self, message):
```

Comment:
Is this function being used?

Reply:
This allows the controller (faber) to tag the active connection (handles the webhook)